lib/internal/cluster/master.js

```diff
@@ -104,8 +104,18 @@ function createWorkerProcess(id, env) {
   workerEnv.NODE_UNIQUE_ID = '' + id;
 
   if (execArgv.some((arg) => arg.match(debugArgRegex))) {
-    execArgv.push(`--inspect-port=${process.debugPort + debugPortOffset}`);
-    debugPortOffset++;
+    let inspectPort;
+    if (cluster.settings.inspectPort) {
```

Comment:
Or should it be `('inspectPort' in cluster.settings)` since we assert types inside the `if`